bigbluebutton-html5/imports/ui/services/livekit/index.ts

```
@@ -26,7 +31,39 @@ export const getLKStats = async (): Promise<Map<string, unknown>> => {
   return statsMap;
 };
 
+export const lkIsCameraSource = (track: TrackPublication | RemoteTrack): boolean => {
+  return track.kind === Track.Kind.Video && track.source === Track.Source.Camera;
+};
+
+export const lkToggleMuteCameras = (mute: boolean): void => {
+  const { videoTrackPublications } = liveKitRoom?.localParticipant;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)